src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -57,7 +57,7 @@ public function renderBlock(FormView $view, $resource, $blockName, array $variab
         // (XDebug limits the level to 100 by default)
         $this->template->displayBlock($blockName, $context, $this->resources[$cacheKey]);
 
-        return ob_get_clean();
+        return (string) ob_get_clean();
```

Comment:
`renderBlock()` must return a string, but `ob_get_clean()` can return false

Reply:
only if ob_start would be missing, isnt it?

Reply:
Yes you are right. `false` cannot happen theoretically here since we `ob_start()` just above, I'll revert. Thanks for spotting it!